src/host.js

```diff
@@ -10,9 +10,12 @@ export default class Host extends Port {
 			throw new Error(`Unable to extract origin from "${src}"`);
 		}
 
-		var parent = document.getElementById(id);
-		if(parent === null) {
-			throw new Error(`Could not find parent node with id "${id}"`);
+		var parent = id;
+		if (typeof id === 'string') {
```

Comment:
@dlockhart : it seems like an odd case to me, but do you think there could be a use case where the caller doesn't have a reference to the container node when constructing the host?  Wondering is perhaps it should be a promise for the container element.  Thoughts?